src/themes/default/components/core/ProductTile.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <div class="product align-center p15">
-    <router-link :to="{ name: 'product', params: { id: product.id, slug: product.slug }}">
+    <router-link :to="{ name: 'product', params: { id: product.id, slug: product.slug, sku: product.configurable_children[0].sku || null }}">
```

Comment:
@kacperknapik it can be misleading the route param should be named like "variant_sku" not SKU because, the key attribute to load product is "id" and product .. has also "sku"; it's misleading, should be named more domain specific